tests/foreman/api/test_activationkey.py

```diff
@@ -491,7 +490,6 @@ def test_positive_fetch_product_content(module_org):
 
 
 @pytest.mark.upgrade
-@skip_if_not_set('fake_manifest')
```

Comment:
for consistency, do you want to include this as well?

Reply:
No, because its stubbed, so there's no functional need for this setting to be set or not.

Reply:
fair enough!